Sources/NIOSSL/SSLPKCS12Bundle.swift

```diff
@@ -158,6 +158,86 @@ public struct NIOSSLPKCS12Bundle: Hashable {
 
 extension NIOSSLPKCS12Bundle: Sendable {}
 
+extension NIOSSLPKCS12Bundle {
+    /// Create a PKCS#12 file containing the given certificates.
+    ///
+    /// The first certificate of the `certificates` array will be considered the "primary" certificate for
+    /// this PKCS#12, and `privateKey` must be its corresponding private key.
+    /// The other certificates included in `certificates`, if any, will be considered as additional
+    /// certificates in the certificate chain.
+    ///
+    /// - Parameters:
+    ///   - certificates: An array of certificates to include in this PKCS#12. Must contain at least one certificate.
+    ///   - privateKey: The private key associated to the first certificate in `certificates`.
+    ///   - passphrase: The password with which to protect this PKCS#12 file.
+    ///   - name: The name to give this PKCS#12 file.
+    /// - Returns: An array of bytes making up the PKCS#12 file.
+    public static func makePKCS12<Bytes: Collection>(
+        certificates: [NIOSSLCertificate],
+        privateKey: NIOSSLPrivateKey,
+        passphrase: Bytes,
+        name: Bytes
```

Comment:
We don't need to provide a name, and I'd argue we shouldn't bother. If we're going to enable this, it should be optional, and needs a different generic type parameter to the passphrase.

Reply:
I'll just get rid of it, don't think it serves any purpose really.